example/main.go

```diff
@@ -1,62 +0,0 @@
-package main
```

Comment:
Shouldn't we keep the example? I guess the example.log can be deleted, but we can keep the example program

Reply:
Sure.